familias/forms.py

```diff
@@ -54,43 +56,78 @@ def __init__(self, *args, **kwargs):
             field.widget.attrs['class'] = 'form-control'
 
 
-class AlumnoForm(ModelForm):
-    """ Model form for Alumno
+class IntegranteModelForm(IntegranteForm):
+    """ This is a general form for any kind of Integrante.
 
-        This is the general form for updating a Alumno.
+    Namely, this works to save and update a regular Integrante,
+    a student, or a tutor.
+    It inherits from the regular model form IntegranteForm and adds
+    the extra fields needed for alumnos and tutors.
     """
-    class Meta:
-        model = Alumno
-        fields = ('integrante',
-                  'numero_sae',
-                  'escuela')
-        widgets = {
-            'integrante': HiddenInput()
-        }
+    # deep copy adding the no relation option.
+    OPCIONES_RELACION = (('', '---------'),) + Tutor.OPCIONES_RELACION
+    escuela = ModelChoiceField(required=False, queryset=Escuela.objects.all())
+    numero_sae = CharField(required=False, max_length=30)
+    relacion = ChoiceField(required=False, choices=OPCIONES_RELACION)
 
     def __init__(self, *args, **kwargs):
         # This adds the class form control to every single input field.
         # Implemented for bootstrap purposes.
-        super(AlumnoForm, self).__init__(*args, **kwargs)
+        super(IntegranteModelForm, self).__init__(*args, **kwargs)
```

Comment:
im not 100% sure but i'm doing it so the things in line 50 are executed, and the __init__ of the forms.ModelForm is executed too.